cldcvr.yaml

```diff
@@ -0,0 +1,31 @@
+environments:
+  - name: Dev
+    provisioner: gcp
+    iacPolicy: "CC default latest"
+    iacRoot: "terraform/dev"
+  
+pipelines:
+  skaffold-deploy:
+    - gke-skaffold@1
```

Comment:
think pipeline config and git clone modules are required as well:

modules:
  * gcp:pipeline-config@1
  * gcp:git-clone@1
  * gcp:gke-skaffold@1

Reply:
I was kind of hoping that VAN-318 would be soon completed (which will automatically add those to the front of the pipeline) - If it isn't or we decide we don't need it for this sprint, I will add those steps here.

Reply:
We decided to push VAN-318 out of this sprint so I added the pipeline-config and git-clone